### ISSUE TITLE
andTestButSkipWhenNull() does not execute the transformation code during test

### ISSUE THREAD

**Issue by thombergs** [remondis-it/remap#25]
The javadoc states 
```
In a future release this method may allow to test the transformation function.
```
so there just seems some functionality missing?

**Comment by schuettec** [comment]
Yes this method should be deleted. Earlier it was designed to assert the transform function to make sure the same instance is configured in the mapper. But since lambdas cannot be compared, this method does not make any sense.

**Comment by schuettec** [comment]
This method should not be deleted but renamed to `andSkipWhenNull()` since we need the information that the skip-when-null-behaviour is expected.

**Comment by schuettec** [comment]
Will be available in version 2.0.0.
